docs: adds notification log docs and updates alerting docs in general (#526)
checkly/checklyhq.com: site/content/docs/alerting/_index.md

## Alert settings

> The [alert settings screen](https://app.checklyhq.com/alerts/settings) gives you a lot of options to tailor when, how and how 

Reviewer: ```suggestion
The [alert settings screen](https://app.checklyhq.com/alerts/settings) gives you the options to tailor when, how and how 
```

often you want to be alerted when a check fails. This is also sometimes referred to as **threshold alerting**. For example:

- Get an alert on the second or third failure.
- Get an alert after 5 minutes of failures.
- Get one or more reminders after a failure is triggered.

> Configuration is really simple. Just remember that you can configure your alert settings at three levels:

Reviewer: ```suggestion
Your alert notifications can be configured at three levels:
```

> 1. **Account** level: This is the default level and applies to all of your check unless you override these settings at the check level.

Reviewer: Or group, or?
